bridge/test/multi-planetary.spec.ts

```diff
@@ -0,0 +1,82 @@
+import Decimal from "decimal.js";
+import { MultiPlanetary } from "../src/multi-planetary";
+
+describe(MultiPlanetary.name, () => {
+    const planetIds = {
+        odin: "0x100000000000",
+        heimdall: "0x100000000001",
+    };
+    const planetVaultAddress = {
+        heimdall: "0xaaaaa6db35d5eff2f0b0758c5ac4c354debaf118",
```

Comment:
no vault for odin?

Reply:
I didn't add odin's valult address because it would not be needed.